Aptoide/appcoins-wallet-android
# [MOB-542] - Added landscape support to OnboardingActivity.

**What does this PR do?**

Allow landscape orientation to OnboardingActivity.
It was created a new layout in layout-land to allow better future customisation.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] activity_onboarding.xml

**How should this be manually tested?**

  Launch wallet in landscape for the first time. It should work properly with rotation.
  Start a payment in landscape before launching the wallet for the first time.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-542](https://aptoide.atlassian.net/browse/MOB-542)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

No


**Code Review Checklist**
